pkg/kn/commands/namespaced_test.go

```diff
@@ -45,12 +46,13 @@ func TestGetNamespaceSample(t *testing.T) {
 	}
 }
 
-// test without setting any namespace
+// test current namespace without setting any namespace
 func TestGetNamespaceDefault(t *testing.T) {
 	testCmd := testCommandGenerator(true)
-	expectedNamespace := "default"
+	expectedNamespace := "current"
 	testCmd.Execute()
-	actualNamespace, err := GetNamespace(testCmd)
+	kp := &KnParams{CurrentNamespace: "current"}
```

Comment:
i think you should add this line to every test case since it's implied that CurrentNamespace() from clientcmd is set. additionally is there an easy way to test GetConfig(). if not in unit test land, can it be done in e2e?